.travis.yml

```diff
@@ -8,18 +8,18 @@ env:
     - TOXENV=django111
     - TOXENV=django20
     - TOXENV=django21
+    - TOXENV=quality
 matrix:
-  include:
-    - python: 2.7
-      env: TOXENV=quality
   exclude:
     - python: 2.7
       env: TOXENV=django20
     - python: 2.7
       env: TOXENV=django21
-  allow_failures:
     - python: 3.6
-    - python: 3.7
+      env: TOXENV=quality
```

Comment:
Why skip the quality check on 3.6?  That's probably the version we'll be running in production next.

Reply:
In my experience the Abstract Syntax Tree (and thus the linter results) of Python 3.6 and 3.7 are nearly identical with two exceptions where Python 3.7 is _stricter_ that 3.6:
1. The strictness around __StopIteration__ that trips up pylint 1.x
2. __async__ and __await__ are keywords which has tripped up Spark, PyTorch, Zappa, etc. but not edX

My sense is that no linting test will pass on 3.7 but fail on 3.6 but we can swap these if you want.  Or we can test quality on all three just to be certain.